### ISSUE TITLE
mdast markdown correction bear

### ISSUE THREAD

**Issue by sils1297** [coala-analyzer/coala#1075]
see https://github.com/wooorm/mdast-lint/issues/29#issuecomment-157879379 , this can reformat markdown which we can use to autocorrect markdown.

**Comment by sils1297** [comment]
Blocked by https://github.com/wooorm/mdast/issues/90 , CCing @wooorm .

**Comment by sils1297** [comment]
not easy because of CI actually, I'm on this one currently

**Comment by sils1297** [comment]
closing in favour of https://github.com/coala-analyzer/coala/issues/1078 which is similar, they have the same problem, us having to use JS in the CI, and both already work locally
